lib/game.rb

```diff
@@ -0,0 +1,96 @@
+require 'pry'
+
+class Game
+
+  attr_accessor :player_1, :player_2, :board
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+    ]
+
+    def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+      @player_1 = player_1
+      @player_2 = player_2
+      @board = board
+      @winner = nil
+    end
+
+    def current_player
+      turn_counter = board.cells.count{|a| a != " "}
```

Comment:
Why not use `Board#turn_count` here instead of re-implementing your logic? Ask the objects questions, that's why we build methods, so our other code is simpler as the object is an encapsulation of the logic.